content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: Customize a Flexible plan
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/
+---
+
+When you create a subscription with a Flexible plan, you customize it to your business needs.
+
+This means you:
+
+- Define deployment settings for your subscription, such as cloud provider, network settings, and so on
+
+- Create at least one database in order to determine cluster size.
+
+- Review the cost estimate for your selections before creating the subscription.
+
+Here's what the process looks like when you start:
+
+&lt;image here>
+
+## Setup settings
+
+The Setup tab specifies general settings for the deployment of your subscription.
+
+Here's what's available:
+
+| Setting | Description |
```

Comment:
Is it our best practice to do left-justified tables? I think they are easier to read in general than center justified.

Reply:
In general, yes...though it's more of an industry and accessibility best-practice.  

For text content.  For numbers and dates, they should be right-aligned for readability.